metaci/release/tasks.py

```diff
@@ -310,6 +320,127 @@ def all_deps_satisfied(
     )
 
 
+PACKAGE_UPLOAD_TASK_PATHS = [
+    "cumulusci.tasks.create_package_version.CreatePackageVersion",
+    "cumulusci.tasks.salesforce.PackageUpload",
+]
+
+
+def get_package_ids_from_build(build: Build) -> Optional[DependencyGraphItem]:
```

Comment:
💭: Some of this logic can replace `release.utils.update_release_from_github()`, which is calling out to github to set  `release.package_version_id` (IMO, 033/04t ids should both be on the release object).